src/handlers/streamHandler.ts

```diff
@@ -99,8 +99,8 @@ export async function handleStreamingMode(response: Response, proxyProvider: str
         writer.close();
     })();
 
-    // Convert GEMINI/COHERE json stream to text/event-stream for non-proxy calls
-    if ([GOOGLE, COHERE].includes(proxyProvider) && responseTransformer) {
+    // Convert GEMINI/COHERE/OLLAMA json stream to text/event-stream for non-proxy calls
+    if ([GOOGLE, COHERE, OLLAMA].includes(proxyProvider) && responseTransformer) {
```

Comment:
This can be removed now as we have added support for OpenAI compatible ollama route. So the response from ollama will always be text/event stream